app/models/comment.rb

```diff
@@ -1,9 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
-  after_create :update_comment_counter
+  # after_create :update_comment_counter
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧